src/Compilers/CSharp/Portable/Symbols/TypeSymbol.cs

```diff
@@ -643,6 +643,13 @@ internal bool ContainsAnnotatedUnconstrainedTypeParameter()
 
         internal abstract TypeSymbol SetUnknownNullabilityForReferenceTypes();
 
+        /// <summary>
+        /// Returns a type with merged nested nullability, or false if nullability
+        /// cannot be merged. The caller is responsible for ensuring the two
```

Comment:
>cannot be merged [](start = 12, length = 16)

Nullability can always be merged. False should indicate that a warning will be required.